src/identity/components/GlobalHeader/AccountDropdown.tsx

```diff
@@ -47,31 +44,25 @@ export const AccountDropdown: FC<Props> = ({
     name: activeAccount.name,
   }
 
-  const isCreateDeleteFlagOn = isFlagEnabled('createDeleteOrgs')
-
   const accountMainMenu = useMemo(
-    () =>
-      [
-        {
-          name: 'Settings',
-          iconFont: IconFont.CogSolid_New,
-          href: `/orgs/${activeOrg.id}/accounts/settings`,
-          enabled: true,
-        },
-        {
-          name: 'Organizations',
-          iconFont: IconFont.OrganizationChart,
-          href: `/orgs/${activeOrg.id}/accounts/orglist`,
-          enabled: isCreateDeleteFlagOn,
-        },
-        {
-          name: 'Billing',
-          iconFont: IconFont.Bill,
-          href: `/orgs/${activeOrg.id}/billing`,
-          enabled: true,
-        },
-      ].filter(menuOption => menuOption.enabled),
-    [activeOrg.id, isCreateDeleteFlagOn]
+    () => [
+      {
+        name: 'Settings',
+        iconFont: IconFont.CogSolid_New,
+        href: `/orgs/${activeOrg.id}/accounts/settings`,
+      },
```

Comment:
There's no need for the 'enabled' distinction here, as all of these tabs are accessible without condition. The global header only exists in cloud, so there's no need for a CLOUD guard either.